### PR TITLE
DOC-9416 PR #116664 - sql/contention: turn on sql.contention.record_serialization_conflicts…

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9416

In insights.md include file, changed sql.contention.record_serialization_conflicts.enabled to default true.

Rendered preview: [Insights page](https://deploy-preview-18507--cockroachdb-docs.netlify.app/docs/v24.1/ui-insights-page.html)

### CODE REVIEW

**Review by xinhaoz — APPROVED** [review]

File: src/current/_includes/v24.1/ui/insights.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Additional information is displayed for the following insight types:
 All transaction executions flagged with this insight type will display a **Transaction with ID {transaction ID} waited on** section which provides details of the blocking transaction execution.
 1. [**Failed Execution**](#failed-execution):
 Certain transaction executions flagged with this insight type will display a **Failed Execution** section with **Conflicting Transaction** and **Conflicting Location** information. The following 3 conditions are required:
-  - The `sql.contention.record_serialization_conflicts.enabled` [cluster setting]({{ link_prefix }}cluster-settings.html) is set to `true`. By default, this is set to `false`.
+  - The [`sql.contention.record_serialization_conflicts.enabled`]({{ link_prefix }}cluster-settings.html#setting-sql-contention-record-serialization-conflicts-enabled) cluster setting is set to `true`(default).
```
Comment (taroface) [inline src/current/_includes/v24.1/ui/insights.md:70]:
```suggestion
  - The [`sql.contention.record_serialization_conflicts.enabled`]({{ link_prefix }}cluster-settings.html#setting-sql-contention-record-serialization-conflicts-enabled) cluster setting is set to `true` (default).
```

File: src/current/_includes/v24.1/ui/insights.md [hunk 2]
```diff
@@ -325,7 +325,7 @@ You can configure [**Workload Insights**](#workload-insights-tab) with the follo
...
 |[`sql.contention.event_store.duration_threshold`]({{ link_prefix }}cluster-settings.html#setting-sql-contention-event-store-duration-threshold)                         | The minimum contention duration to cause contention events to be collected into the `crdb_internal.transaction_contention_events` table.                                                      | [Transaction executions](#transaction-executions-view)               |
-|`sql.contention.record_serialization_conflicts.enabled` | enables recording `40001` errors with conflicting txn meta as `SERIALIZATION_CONFLICT` contention events into `crdb_internal.transaction_contention_events` | [Transaction executions](#transaction-executions-view)               |
+|[`sql.contention.record_serialization_conflicts.enabled`]({{ link_prefix }}cluster-settings.html#setting-sql-contention-record-serialization-conflicts-enabled) | enables recording `40001` errors with conflicting txn meta as `SERIALIZATION_CONFLICT` contention events into `crdb_internal.transaction_contention_events`<br><br>Default: `true` | [Transaction executions](#transaction-executions-view)               |
```
Comment (taroface) [inline src/current/_includes/v24.1/ui/insights.md:328]: I don't know what "conflicting txn meta" means, but here is an educated guess.

```suggestion
|[`sql.contention.record_serialization_conflicts.enabled`]({{ link_prefix }}cluster-settings.html#setting-sql-contention-record-serialization-conflicts-enabled) | enables recording `40001` errors, along with metadata about conflicting transactions, as `SERIALIZATION_CONFLICT` events into `crdb_internal.transaction_contention_events`.<br><br>**Default:** `true` | [Transaction executions](#transaction-executions-view)               |
```

**Review by taroface — APPROVED** [review]
LGTM with 1 nit and 1 fairly strong suggestion

File: src/current/_includes/v24.1/ui/insights.md (hunk 1 above)
Comment (florence-crl) [reply src/current/_includes/v24.1/ui/insights.md:70]: updated

File: src/current/_includes/v24.1/ui/insights.md (hunk 2 above)
Comment (florence-crl) [reply src/current/_includes/v24.1/ui/insights.md]: thanks for the suggestion! updated.

**Pull request merged**